content/codespaces/about-codespaces.md

```diff
@@ -0,0 +1,72 @@
+---
+title: About Codespaces
+intro: '{% data variables.product.prodname_codespaces %} is a configurable online development environment, hosted by {% data variables.product.prodname_dotcom %} and powered by {% data variables.product.prodname_vscode %}, that allows you to develop entirely in the cloud.'
```

Comment:
```
> ```suggestion
intro: '{% data variables.product.prodname_codespaces %} is a configurable online development environment, hosted by {% data variables.product.prodname_dotcom %} and powered by {% data variables.product.prodname_vscode %}, that allows you to develop entirely in the cloud.'
```
If Music Label=Name
    Print(Strezless Musick Productionz) 
If Artist=Name
     Print(Sin Estres)
```

Reply:
+Omar Dejesu Orrantia